backend/src/test/java/de/jankahle/capstone/service/RecipeServiceTest.java

```diff
@@ -0,0 +1,52 @@
+package de.jankahle.capstone.service;
+
+import de.jankahle.capstone.db.RecipeMongoDB;
+import de.jankahle.capstone.model.Ingredient;
+import de.jankahle.capstone.model.Recipe;
+import de.jankahle.capstone.model.RecipeDto;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class RecipeServiceTest {
+
+    private final RecipeMongoDB recipeMongoDB = mock(RecipeMongoDB.class);
```

Comment:
mock is not used in test, you may want to remove it or include a mock response within your test

Reply:
It actually IS used in the test, but the implementation does not matter in the context of the test as the return value of the recipeMongoDB.save method is not used in any way. Therefore we do not need to mock with when.
I'll add a verify to check if the save method is invoked though.